### PR TITLE
Update SecurityCardManagement proposal

### PR DESCRIPTION
**Pull request by Oxydixi**

### CODE REVIEW

**Review by sertony — APPROVED** [review]

File: SecurityCardManagement/README.md [hunk 1]
```diff
@@ -15,26 +15,27 @@ Allows to manage security card.
...
-	answerId: uint32  - function id of result callback
+	answerId: uint32 - function id of result callback
+	sn: uint192 - card serial number
 
-returns: 
+returns:
 
 	h2: uint256 - H2 hash
 	h3: uint256 - H3 hash
+	sn: uint192 - card serial number
```
Comment (Keshoid) [inline SecurityCardManagement/README.md:29]: Why do we need to return `sn` here? `sn` is an input argument.

File: SecurityCardManagement/README.md [hunk 2]
```diff
@@ -79,12 +80,21 @@ returns:
...
+
+`createKeyForHmac` - create secret key for HMAC SHA256. This key is saved into Android keystore or iOS keychain and then is used by the app to sign APDU commands data fields. 
+
+arguments: 
+
+	answerId: uint32  - function id of result callback
+	p1:       bytes   - authentication password (128 bytes) 
+	cs:       bytes   - common secret(32 bytes)
+	sn:       uint192 - card serial number
+
```
Comment (Keshoid) [inline SecurityCardManagement/README.md:96]: Mark that the function returns nothing like:
```
returns:
    void
```

Comment (Keshoid) [reply SecurityCardManagement/README.md:96]: or maybe it can return status like `success` or fail reason.

**Review by Keshoid — CHANGES_REQUESTED** [review]

File: SecurityCardManagement/README.md (hunk 1 above)
Comment (sertony) [reply SecurityCardManagement/README.md:29]: It'd rather should be removed from arguments

File: SecurityCardManagement/SecurityCardManagement.abi.json [hunk 3]
```diff
@@ -1,78 +1,90 @@
...
-}
+{
+	"ABI version": 2,
+	"header": ["time"],
+	"functions": [
+		{
+			"name": "getBlockHashes",
+			"inputs": [
+				{"name":"answerId","type":"uint32"},
+				{"name":"_sn","type":"uint192"}
```
Comment (sertony) [inline SecurityCardManagement/SecurityCardManagement.abi.json:9]: Why does it have underscore prefix? (_sn)

File: SecurityCardManagement/README.md (hunk 1 above)
Comment (Oxydixi) [reply SecurityCardManagement/README.md:29]: done

File: SecurityCardManagement/README.md (hunk 2 above)
Comment (Oxydixi) [reply SecurityCardManagement/README.md:96]: done

**Review by sertony — APPROVED** [review]

**Review by Keshoid — APPROVED** [review]
LGTM

**Pull request merged**